conf/eslint.json

```diff
@@ -88,10 +88,10 @@
         "no-return-assign": "off",
         "no-script-url": "off",
         "no-self-assign": "error",
-        "no-self-compare": "off",
+        "no-self-compare": "error",
```

Comment:
Please revert all these changes, this affects everyone using `eslint:recommended`. If you're proposing to enable these rules for just the ESLint project, please make changes in `packages/eslint-config-eslint/default.yml`.